localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -410,6 +410,9 @@
         <trans-unit id="msgRefreshTokenError">
             <source xml:lang="en">Error when refreshing token</source>
         </trans-unit>
+        <trans-unit id="msgAzureCredStoreSaveFailedError">
+            <source xml:lang="en">Keys for token cache could not be saved in credential store, this may cause Azure access token persistence issues and connection instabilities. Its likely that VS Code has reached credential storage limit on Windows, please clear atleast 2 credentials created by VS Code in Windows Credential Manager and reload.</source>
```

Comment:
```suggestion
            <source xml:lang="en">Keys for token cache could not be saved in credential store, this may cause Azure access token persistence issues and connection instabilities. It's likely that VS Code has reached credential storage limit on Windows, please clear at least 2 credentials created by VS Code in Windows Credential Manager and reload.</source>
```